qiskit_experiments/visualization/drawers/mpl_drawer.py

```diff
@@ -454,6 +457,75 @@ def textbox(
         )
         text_box_handler.set_bbox(bbox_props)
 
+    def _series_names_to_cmap(self, series_names: List[str]) -> Tuple[Colormap, Dict[str, float]]:
+        # Remove duplicates from series_names, just in-case. Use dict.fromkeys to preserve order and
+        # remove duplicates.
+        unique_series_names = list(dict.fromkeys(series_names))
+
+        # Generate list of colours by calling querying series_params and self._get_default_color(name).
+        colours = []
+        for series_name in unique_series_names:
+            series_params = self.figure_options.series_params.get(series_name, {})
+            colour = series_params.get("color", self._get_default_color(series_name))
+            colours.append(colour)
+
+        # Create CMap.
+        cmap = LinearSegmentedColormap.from_list(
+            "SeriesMap",
+            colours,
+        )
+
+        # Create a dictionary to lookup the floating-point value for each series name.
+        series_cmap_idx = dict(
+            zip(unique_series_names, np.linspace(0, 1, len(unique_series_names)))
+        )
+
+        return cmap, series_cmap_idx
+
+    def image(
+        self,
+        data: np.ndarray,
+        extent: Optional[ExtentTuple] = None,
+        name: Optional[str] = None,
+        label: Optional[str] = None,
+        cmap: Optional[Union[str, Any]] = None,
+        cmap_use_series_colors: bool = False,
+        colorbar: bool = False,
+        **options,
+    ):
```

Comment:
`extent` originates with mpl, yes. It is the rectangular region within which the image must be drawn. Given that most people use Matplotlib to create figures, and that `extent` has effectively the same role for `drawer.image()`, I figured it would be appropriate to use the same name. But I'm open to changing it.

Reply:
Ok. Given the connection with mpl and the widespread usage of mpl I am fine with this name.